javaparser-core/src/main/java/com/github/javaparser/JavaToken.java

```diff
@@ -417,6 +418,8 @@ public enum Kind {
 
         public static Kind valueOf(int kind) {
             switch(kind) {
+                case 151:
```

Comment:
That's not right. I'm not sure whether the valueOf method is automatically generated by the TokenKindGenerator.

Reply:
Good catch